test/spec/windshaft/error.mock.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  responseText: '{ "errors": ["Postgis Plugin: ERROR:  transform: couldnt project point (242 611 0): latitude or longitude exceeded limits."]}'
```

Comment:
Where is this used? I can't see it anywhere

Reply:
engine.spec.js:7

Reply:
the file was closed because it was too big for github 😂

Reply:
Tests, tests, tests :green_heart: